apko-build/action.yaml

```diff
@@ -47,6 +47,16 @@ inputs:
     required: false
     default: false
 
+  automount-src:
+    description: |
+      If this directory is found, create a copy at automount-dest (inside container)
+    default: ${{ github.workspace }}/.apko-automount
```

Comment:
Curious, what's the rationale for making the default source a dotfile?

Reply:
i... am not so sure 😅

ok to change this